yb-voyager/cmd/import.go

```diff
@@ -166,8 +166,7 @@ func registerImportDataFlags(cmd *cobra.Command) {
 			"false - to not truncate splits after importing (required for debugging)")
 	cmd.Flags().MarkHidden("truncate-splits")
 
-	cmd.Flags().BoolVar(&liveMigration, "live-migration", false,
-		"set this flag to true to enable streaming data from source to target database")
+	cmd.Flags().StringVar(&importType, "import-type", "snapshot", "import type: snapshot-only, changes-only, snapshot-and-changes")
```

Comment:
Default value is incorrect.

As suggested above define and use constants: SNAPSHOT_ONLY, CHANGES_ONLY, and SNAPSHOT_AND_CHANGES.